server/src/main/java/com/indexdata/reservoir/server/HealthCheck.java

```diff
@@ -0,0 +1,30 @@
+package com.indexdata.reservoir.server;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.ext.web.Router;
+import org.folio.tlib.RouterCreator;
+import org.folio.tlib.postgres.TenantPgPool;
+
+public class HealthCheck implements RouterCreator {
+  @Override
+  public Future<Router> createRouter(Vertx vertx) {
+    Router router = Router.router(vertx);
+    router.route(HttpMethod.GET, "/admin/health").handler(ctx -> {
```

Comment:
@adamdickmeiss how about we put this in addition to the /adnin/health as /healthz so the chart uses it but we don't mess with whatever Okapi does?